grpc/src/main/java/com/linecorp/armeria/server/grpc/ArmeriaServerCall.java

```diff
@@ -303,13 +303,19 @@ private void doClose(Status status, Metadata unusedGrpcMetadata) {
             trailersObj = trailers;
         }
         try {
-            res.write(trailersObj);
-            res.close();
+            if (res.tryWrite(trailersObj)) {
```

Comment:
I'm curious that don't we need to try to release `trailersObj` if `tryWrite` fails?

Reply:
`tryWrite()` releases the given object if write fails. :-)

Reply:
I see. I missed that. Thank you!